securesystemslib/formats.py

```diff
@@ -79,7 +79,7 @@
 import six
 
 from securesystemslib import exceptions
-import securesystemslib.schema as SCHEMA
+from securesystemslib import schema as SCHEMA
```

Comment:
I guess you kept `as SCHEMA` to make the commit less invasive? This might be something to note in the commit message.

Reply:
Let me know if you want to reword the commit message. Otherwise, I'll just go ahead and merge. :)

Reply:
Yes exactly. And I think I can live with that message if you can :)